### PR TITLE
[FIX] investor_wallet_platform_base: Users can read structures that have shares in their structure

### PR DESCRIPTION
**Pull request by carmenbianca**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=7372&action=479&model=project.task&view_type=form&menu_id=536

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: investor_wallet_platform_base/security/iwp_security.xml [hunk 1]
```diff
@@ -43,6 +43,29 @@
             </field>
         </record>
 
+        <record
+            id="res_partner_emc_user_read_structures_with_shares_rule"
```
Comment (victor-champonnois) [inline investor_wallet_platform_base/security/iwp_security.xml:47]: Maybe it's the occasion to remove the "emc" from the id ? :)

File: investor_wallet_platform_base/security/iwp_security.xml [hunk 2]
```diff
@@ -43,6 +43,29 @@
...
+            model="ir.rule"
+        >
+            <field
+                name="name"
+            >res_partner_emc_user_read_structures_with_shares_rule</field>
```
Comment (victor-champonnois) [inline investor_wallet_platform_base/security/iwp_security.xml:52]: idem here

**Review by victor-champonnois — APPROVED** [review]
I am not competent with Odoo security, but if it pass the functional tests, I think it is OK.

**Comment by victor-champonnois** [comment]
@carmenbianca Also do you think that a test case is needed ?

File: investor_wallet_platform_base/security/iwp_security.xml (hunk 1 above)
Comment (carmenbianca) [reply investor_wallet_platform_base/security/iwp_security.xml:47]: I think we can do this during migration to v14. I don't know if the old security records are deleted if we rename them here. I could test all that, but there's so many PRs that need merging, and this repo is really only used by one client.

**Comment by carmenbianca** [comment]
No idea how to test security stuff. It's probably fine.

**Pull request merged**